libc-test/semver/windows.txt

```diff
@@ -354,4 +362,4 @@ write
 wrmdir
 wsetlocale
 wstat
-wutime
+wutime
```

Comment:
Any idea why it shows a diff here? (Nothing showing up on the UI)

Reply:
Looks like a new line was added to the end of the file.

Reply:
Oh good eye, yeah the trailing newline was removed. I thought GH usually showed a red circle at the end of the file when a newline is missing. @arihant2math mind putting that back?

(also Chris I wouldn't mind a second set of eyes on this from a Windows expert)

Reply:
Done